recipes/rill/build.sh

```diff
@@ -0,0 +1,10 @@
+pushd cli
```

Comment:
Make this a bash script in strict mode.
```suggestion
#!/bin/bash

set -exuo pipefail

pushd cli
```